### PR TITLE
llvm10

### PR DESCRIPTION
**Pull request by thewilsonator**
Fix compilation with head of tree

### CODE REVIEW

File: driver/toobj.cpp [hunk 1]
```diff
@@ -47,6 +47,14 @@
 #include <cstddef>
 #include <fstream>
 
+#if LDC_LLVM_VER < 1000
+using CGFT_AssemblyFile = llvm::TargetMachine::CGFT_AssemblyFile;
+using CGFT_ObjectFile = llvm::TargetMachine::CGFT_ObjectFile;
```
Comment (JohanEngelen) [inline driver/toobj.cpp]: what are these for llvm >= 10 ?

Comment (thewilsonator) [reply driver/toobj.cpp]: Nothing? Maybe we have a using hiding something somewhere, but it compiles for me as is.

File: driver/toobj.cpp [hunk 2]
```diff
@@ -57,15 +63,15 @@ namespace {
...
+                   CodeGenFileType fileType) {
   using namespace llvm;
 
   const ComputeBackend::Type cb = getComputeTargetType(&m);
 
   if (cb == ComputeBackend::SPIRV) {
 #ifdef LDC_LLVM_SUPPORTED_TARGET_SPIRV
     IF_LOG Logger::println("running createSPIRVWriterPass()");
-#if LDC_LLVM_VER >= 900
+#if LDC_LLVM_VER == 900
```
Comment (kinke) [inline driver/toobj.cpp]: 901 is due in a few weeks.

Comment (thewilsonator) [reply driver/toobj.cpp]: Fixed

File: driver/toobj.cpp (hunk 1 above)
Comment (thewilsonator) [reply driver/toobj.cpp]: Derp its the using below.

**Comment by thewilsonator** [comment]
I think this is good to go now.

**Pull request merged**